examples/server-enum.py

```diff
@@ -50,7 +50,7 @@ def enum_to_stringlist(a_enum):
 
     # 1.
     # Create Enum Type
-    myenum_type = enums.add_subtype(nsidx, 'MyEnum', ua.NodeClass.DataType)
+    myenum_type = enums.create_data_type(nsidx, 'MyEnum')
```

Comment:
can you also squash line and line 64 into one?

Tip: use `server.create_custom_data_type` (It is possible that method should be extended)